internal/csi-addons/networkfence/fencing_test.go

```diff
@@ -68,6 +68,11 @@ func TestFetchIP(t *testing.T) {
 			expectedIP:  "172.21.9.34",
 			expectedErr: false,
 		},
+		{
+			clientInfo:  "client.4305 2001:0db8:85a3:0000:0000:8a2e:0370:7334:0/422650892",
```

Comment:
How did you confirm that Ceph uses this format with IPv6 addresses?

Reply:
From the official documentation of ceph
As the inst listed in ceph tell mds.0 client ls for the IPv4 address,
'"inst": "client.4305 172.21.9.34:0/422650892", 
So IPv6 should also be the same only like the client id and then..
I'll check with ceph for more clarity on this.

Reply:
Checked with Xiubo and Kotresh, the given syntax is fine!